### PR TITLE
Add 8 new k6 unit tests

### LINKED ISSUE #5104: Add remaining missing rest api performance tests to k6 test suite
### Problem

The following rest endpoints are missing from the k6 spec tests.

```
/contracts/{id}/state
/contracts/result/{transactionId}
/contracts/result/{transactionId}/actions
/contracts/result/logs
/network/exchangerate
/network/fees
/network/nodes
/transactions/{id}/stateproof
```

### Solution

Add tests for the above mentioned rest endpoints to the k6 spec.

A part of this was done as part of https://github.com/hashgraph/hedera-mirror-node/issues/4961 for reference

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by MarcKriguerAtHedera**
(those not already covered in recent previous k6 work).

**Description**:

This PR adds 8 k6 tests to cover API methods not yet covered.

**Related issue(s)**:

Fixes #5104

**Notes for reviewer**:
6 of the new tests were successful for 100% of the k6 runs, one for 93%, but the "Contracts Results id Actions" test was not able to get a non-empty list of actions.  See details as a comment on the ticket.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (k6)

### CODE REVIEW

File: hedera-mirror-test/k6/src/rest/test/networkNodes.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import {isValidListResponse} from "./common.js";
+import {setupTestParameters} from "./bootstrapEnvParameters.js";
+
+const urlTag = '/network/nodes';
+
+const {options, run} = new TestScenarioBuilder()
+  .name('networkNodes') // use unique scenario name among all tests
+  .tags({url: urlTag})
+  .request((testParameters) => {
+    const url = `${testParameters['BASE_URL']}${urlPrefix}${urlTag}`;
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/rest/test/networkNodes.js:34]: Add limit=25 to increase to max allowed

File: hedera-mirror-test/k6/src/rest/test/contractsResultsLogs.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import {isValidListResponse} from "./common.js";
+import {setupTestParameters} from "./bootstrapEnvParameters.js";
+
+const urlTag = '/contracts/results/logs';
+
+const {options, run} = new TestScenarioBuilder()
+  .name('contractsResultsLogs') // use unique scenario name among all tests
+  .tags({url: urlTag})
+  .request((testParameters) => {
+    const url = `${testParameters['BASE_URL']}${urlPrefix}/contracts/results/logs`;
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/rest/test/contractsResultsLogs.js:34]: Add DEFAULT_LIMIT

File: hedera-mirror-test/k6/src/rest/test/contractsResultsIdActions.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import {isValidListResponse} from "./common.js";
+import {setupTestParameters} from "./bootstrapEnvParameters.js";
+
+const urlTag = '/contracts/results/{id}/actions';
+
+const {options, run} = new TestScenarioBuilder()
+  .name('contractsResultsIdActions') // use unique scenario name among all tests
+  .tags({url: urlTag})
+  .request((testParameters) => {
+    const url = `${testParameters['BASE_URL']}${urlPrefix}/contracts/results/${testParameters['DEFAULT_CONTRACT_RESULT_HASH']}/actions`;
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/rest/test/contractsResultsIdActions.js:34]: Add DEFAULT_LIMIT

File: hedera-mirror-test/k6/src/rest/test/contractsResultsId.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+import {isSuccess} from "./common.js";
+import {setupTestParameters} from "./bootstrapEnvParameters.js";
+
+const urlTag = '/contracts/results/{id}';
+
+const {options, run} = new TestScenarioBuilder()
+  .name('contractsResultsId') // use unique scenario name among all tests
+  .tags({url: urlTag})
+  .request((testParameters) => {
+    const url = `${testParameters['BASE_URL']}${urlPrefix}/contracts/results/${testParameters['DEFAULT_CONTRACT_RESULT_HASH']}`;
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/rest/test/contractsResultsId.js:34]: Add DEFAULT_LIMIT

File: hedera-mirror-test/k6/src/rest/test/contractsIdState.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+import {isValidListResponse} from "./common.js";
+import {setupTestParameters} from "./bootstrapEnvParameters.js";
+
+const urlTag = '/contracts/{id}/state';
+
+const {options, run} = new TestScenarioBuilder()
+  .name('contractsIdState') // use unique scenario name among all tests
+  .tags({url: urlTag})
+  .request((testParameters) => {
+    const url = `${testParameters['BASE_URL']}${urlPrefix}/contracts/${testParameters['DEFAULT_CONTRACT_EVM_ADDRESS']}/state`;
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/rest/test/contractsIdState.js:34]: Add DEFAULT_LIMIT

File: hedera-mirror-test/k6/src/lib/parameters.js [hunk 6]
```diff
@@ -149,16 +151,27 @@ export const computeAccountParameters = (configuration) =>
...
-    DEFAULT_CONTRACT_ID: contract.contract_id,
-    DEFAULT_CONTRACT_TIMESTAMP: log.timestamp,
-  });
-  return computeProperties(['DEFAULT_CONTRACT_ID', 'DEFAULT_CONTRACT_TIMESTAMP'], () =>
-    getPropertiesForEntity(configuration, extractProperties, {
+  const contractProperties = computeProperties(['DEFAULT_CONTRACT_ID', 'DEFAULT_CONTRACT_TIMESTAMP', 'DEFAULT_CONTRACT_EVM_ADDRESS'], () => {
+    const extractProperties = (contract, log) => ({
+      DEFAULT_CONTRACT_ID: contract.contract_id,
+      DEFAULT_CONTRACT_TIMESTAMP: log.timestamp,
+      DEFAULT_CONTRACT_EVM_ADDRESS: contract.evm_address,
```
Comment (steven-sheehy) [inline hedera-mirror-test/k6/src/lib/parameters.js:158]: We don't need DEFAULT_CONTRACT_EVM_ADDRESS, can just use DEFAULT_CONTRACT_ID for contract state retrieval

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-test/k6/src/rest/test/networkNodes.js (hunk 1 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-test/k6/src/rest/test/networkNodes.js:34]: Done.

File: hedera-mirror-test/k6/src/rest/test/contractsResultsLogs.js (hunk 2 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-test/k6/src/rest/test/contractsResultsLogs.js:34]: Done.

File: hedera-mirror-test/k6/src/rest/test/contractsResultsIdActions.js (hunk 3 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-test/k6/src/rest/test/contractsResultsIdActions.js:34]: Done.

File: hedera-mirror-test/k6/src/rest/test/contractsResultsId.js (hunk 4 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-test/k6/src/rest/test/contractsResultsId.js:34]: Done.

File: hedera-mirror-test/k6/src/rest/test/contractsIdState.js (hunk 5 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-test/k6/src/rest/test/contractsIdState.js:34]: Done.

File: hedera-mirror-test/k6/src/lib/parameters.js (hunk 6 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-test/k6/src/lib/parameters.js:158]: Done.  (Removed.)

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by edwin-greene — DISMISSED** [review]
Looks good!

**Comment by MarcKriguerAtHedera** [comment]
I was able to fix the merge conflicts and get all the tests running again.

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**